pages/common/bash.md

```diff
@@ -28,6 +28,10 @@
 
 `bash -s`
 
+- Stop bash argument processing and pass all subsequent arguments directly to the script being executed:
+
+`bash {{path/to/script}} -- {{arguments}}`
```

Comment:
path/to/file.sh to be consistent

Reply:
This is still the same as `bash -c "..."`, and it also seems to be an unneeded example, at least based on my experiments with it:

Here, `-c` and `-i` are `bash` options, bu they still get ignored by `bash` and get passed to `x.sh`:

![image](https://user-images.githubusercontent.com/14198070/63641103-29870d00-c6a9-11e9-8f1c-c1c7d72129b4.png)

Reply:
Ohh I see, I hadn't read the man page. But now it is clear. Yes you are right, I think we need to adjust our description of `-c` to properly reflect this. 

> -c        If the -c option is present, then commands are read from the first non-option argument command_string.  If there are argu‐
                 ments after the command_string, the first argument is assigned to $0 and any remaining arguments are assigned to the posi‐
                 tional parameters.  The assignment to $0 sets the name of the shell, which is used in warning and error messages.

Reply:
![image](https://user-images.githubusercontent.com/14198070/63655420-01baa680-c788-11e9-8139-85f72fc5e82c.png)

Don't think the description for the `-c` command needs any change. If anything, like I said in my first review above, I would add `--with args`, making the example:

    bash -c "{{command --with args}}"

Reply:
After thinking about it, I'm guessing that the idea with the `--` one is to avoid nasty issues with quotes. Hrm :thinking:

Reply:
When I read the description, I thought it is only used for executing commands like echo and date, but did not realize that this option was actually about consuming arguments properly.

Updating the example is also necessary. But the description should clarify when should we use "bash -c "{{something}}"" vs "bash {{something}}". The current description just says "execute a command" vs "execute a file", which is why I thought `-c` is only for executing oneliner commands.

Reply:
@agnivade  it **is** only for commands. In my last example, `bash -c "./x.sh ..."` only works because `x.sh` is marked executable and has the proper shebang (note the `./` prefix, since the current directory is not in my `$PATH`). Something like `bash -c "x.sh ..."` would not work (even if the file is marked as executable), because the current folder is not in the `$PATH`.

In other words, `-c` is **not** to execute files *and* commands, it's only to execute bash commands, for example:

    $ bash -c "if [ 0 -eq 0 ]; then echo 1; else echo 2; fi"
    1

Reply:
I see. Thanks. Then is there a need to add `--` to the `-c` example ? Because if the command is enclosed within quotes then it is already safe from intereference with the other bash arguments.

Reply:
@sbrl granted that, but I don't really see the point since these two invocations are the same:

    $ bash ./script.sh arg1 arg2 arg3
    $ bash ./script.sh -- arg1 arg2 arg3

This PR is basically just a "special case" for the second example really.

Reply:
@agnivade IMHO no, I see no need to add `--` for the `-c` example.

Reply:
@mebeim @agnivade @sbrl But this option is necessary when you want to pass arguments to a script that is piped into bash, for example:
``` bash
$ curl -fsSL https://git.io/n-install | bash -s -- {{arugments}}
```
If you don't add the `--` option then curl will recieve the arguments, not the script that is piped into bash.

Reply:
The problem is I am not sure if I could use this curl command as the example.

Reply:
As I already said, that does not seem to be needed for me. And besides, this is a *very different* example than the one you are proposing to add in this PR.

![image](https://user-images.githubusercontent.com/14198070/63670591-71a74c00-c7dd-11e9-8698-87f3219b6bb9.png)

What version of bash are you using?

Reply:
Hrm. It does seem to be a special use-case, so perhaps it's not a good fit for tldr-pages, since we prefer to focus on the most common use-cases. I'm not sure. @waldyrious?

Reply:
Waaay behind schedule :sweat_smile:, but reading the entire discussion (and special thanks to @mebeim for the thorough investigation), I agree that we would be entering tricky waters that are not a good fit for tldr's beginner-focused approach. Especially since the page is already sizeable with the other examples.